pallets/baby-liminal/src/lib.rs

```diff
@@ -46,6 +53,12 @@ pub mod pallet {
         /// Limits how many bytes proof or public input can have.
         #[pallet::constant]
         type MaximumDataLength: Get<u32>;
+
+        /// Deposit amount for storing a verification key
+        ///
+        /// Will get locked and returned upon deleting the key by the owner
+        #[pallet::constant]
+        type VerificationKeyDepositAmount: Get<VerificationKeyDeposit<Self>>;
```

Comment:
I think that we should have something like `DepositPerByte` like pallets contracts and mulitsig have - since we allow for different proof systems, the key lengths may vary from each other. However, in order to avoid problems with such cost changing over time, we should also store what exactly _was_ deposit for every key independently and return that value after key removal.

Reply:
done